.github/workflows/rust.yml

```
@@ -41,6 +41,133 @@
+        run: cargo test --workspace --no-default-features
+      - name: Build workspace (release mode)
+        run: cargo build --workspace --release
+      
+  clippy:
+    name: Clippy (Non-strict)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Crate
+        uses: actions/checkout@v4
+      - name: Setup Rust toolchain
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)